speech/google/cloud/speech/sample.py

```diff
@@ -110,6 +118,17 @@ def sample_rate(self):
         return self._sample_rate
 
     @property
+    def stream(self):
+        """Stream the content when it is a file-like object.
+
+        :rtype: file
+        :returns: File like object to stream.
+        """
+        if getattr(self._stream, 'read', None) is None:
+            return None
+        return self._stream
```

Comment:
ISTM we can safely just return `self._stream`? "Trust" that the user passed a file-like. @jonparrott WDYT?